pkg/chain/ethereum/ethereum.go

```diff
@@ -22,15 +22,19 @@ import (
 var logger = log.Logger("keep-chain-ethereum")
 
 // ThresholdRelay converts from ethereumChain to beacon.ChainInterface.
-func (ec *ethereumChain) ThresholdRelay() relaychain.Interface {
+func (ec *ethereumChain) ThresholdRelay() relayChain.Interface {
 	return ec
 }
 
 func (ec *ethereumChain) GetKeys() (*operator.PrivateKey, *operator.PublicKey) {
 	return operator.EthereumKeyToOperatorKey(ec.accountKey)
 }
 
-func (ec *ethereumChain) GetConfig() *relaychain.Config {
+func (ec *ethereumChain) Signing() chain.Signing {
```

Comment:
Where are we using this function?

Reply:
Looks like we use it in `beacon.go` which implements an interface `Signing` from `chain.go`

Reply:
It's fetched here:
https://github.com/keep-network/keep-core/blob/b6993e6dcf3bfb3c48a68e6e899954c4a220aeef/pkg/beacon/beacon.go#L52

and used later to generate relay entry.

There are also tests:
- `signing_test.go`
- `states_test.go`
- `membership_validator_test.go`
- `dkgtest.go`